.github/workflows/main.yml

```
@@ -48,7 +48,7 @@ jobs:
 
   push-to-docker:
     needs: build-test-cache
-    if: always() && ((contains(needs.build-test-cache.result, 'success') && !contains(needs.integration.outputs.started, 'true')) || (contains(needs.integration.result, 'success'))) && !github.event.act
+    if: always() && ((contains(needs.build-test-cache.result, 'success') && !contains(needs.integration.outputs.started, 'true')) || (contains(needs.integration.result, 'success'))) && !github.event.act && github.actor != 'dependabot[bot]'
```

is there a better wat to do this giant if statement?

---

I think it needs to be like this because of the OR statement
```suggestion
    if: always() && github.actor != 'dependabot[bot]' && ((contains(needs.build-test-cache.result, 'success') && !contains(needs.integration.outputs.started, 'true')) || (contains(needs.integration.result, 'success'))) && !github.event.act && github.actor != 'dependabot[bot]'
```

---

No I don;t think so. Your change is correct